account_chart_update/wizard/wizard_chart_update.py

```
@@ -503,9 +505,10 @@ def find_repartition_by_templates(
                     )
                 )
 
-        # Mark to be removed the lines not found
-        remove_ids = [x for x in current_repartition.ids if x not in existing_ids]
-        result += [(2, x) for x in remove_ids]
+        if tax.amount_type != "group":
```

We should prevent the creation of such default lines instead of masking this change, isn't it?

---

What do you mean by "masking this change"?

---

ping @pedrobaeza

---

On tax of type `group`, we mask any problem/update. The solution is to prevent the creation of such default lines.

---

But the creation of those reparation lines is due to the default value set in Odoo standard, do you think Odoo would accept the change (specially in 14.0)?

---

I think there are several ways of doing them from here:

- Inhibit them through context and override.
- Delete them later.
- Be creative!

---

@pedrobaeza What for the databases that already have the groups with this repartition lines created?

I think the fix is quite clear, we don't really care of repartition lines for tax groups as they don't have any effect or any behavior. It is fair to skip tax groups in a `find_repartition_by_templates` when we know repartition does not apply to groups.

What problem do you envision with this fix? It has been long ago since I propose this and it is working fine for me and so it seems for @lk-eska

---

OK, let's go with this.